docs/dev-setup.md

```diff
@@ -174,6 +174,32 @@ refer to
 [the development setup guide of CAPM3](https://github.com/metal3-io/cluster-api-provider-metal3/blob/main/docs/dev-setup.md#tilt-for-dev-in-capm3)
 and specially the [Baremetal Operator Integration](https://github.com/metal3-io/cluster-api-provider-metal3/blob/main/docs/dev-setup.md#including-baremetal-operator-and-ip-address-manager)
 
+### Making (virtual) BareMetalHosts with Tilt interface
+
+After Tilt is up it is possible to make baremetalhosts by pressing a button in the Tilt localhost interface (right upper corner). This is currently only supported for Unix based systems. This button runs the content of file
+
+```sh
+tools/bmh_test/create_bmh.sh <NAME> <VBMC_PORT>
+```
+
+and adds the values given to the button as arguments. Controlplane host can be created with
```

Comment:
This works but was a bit hard to understand I think. Or maybe I am using it wrong. :sweat_smile:  Could you make it so the drop down where we add the value also has a button for creating?
Current flow:

1. drop down and fill in values
2. "go back" and press the +

What I was thinking is like in CAPI:

1. drop down and fill in values
2. below the values press "Create"

Reply:
I wasn't able to find a way to make this work in the second way. For what I was able to find Tilt doesn't support adding buttons to the drop down.

Reply:
Ok, I think I was confusing myself. It looks different in different places apparently. The button is good :+1: